### PR TITLE
Cookies Banner Component

### PR DESCRIPTION
**Pull request by atomiomi**

### CODE REVIEW

File: packages/components/src/cookies_banner/index.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  children,
+  content,
+  GTMUrl,
+  insertGTM,
+  dismissed,
+  onScriptCreate,
+  ...rest
+}) => {
+  if (insertGTM) return <Script url={GTMUrl} onCreate={onScriptCreate} />;
+  if (dismissed) return null;
```
Comment (suprMax) [inline packages/components/src/cookies_banner/index.jsx:20]: this is probably not necessary/can be moved outside `condition && <Component />`

File: packages/components/src/cookies_banner/index.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { text, boolean, withKnobs } from '@storybook/addon-knobs';
+
+import CookiesBanner from './index';
+
+export default { title: 'CookiesBanner', component: CookiesBanner, decorators: [withKnobs] };
+
+export const Default = () => (
+  <BrowserRouter>
```
Comment (suprMax) [inline packages/components/src/cookies_banner/index.stories.jsx:10]: make this `<BrowserRouter>` wrapping a default? @peterhass thoughts?

**Review by suprMax — APPROVED** [review]

File: packages/components/src/cookies_form/index.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+}) => {
+  const handleValidSubmit = ({ tracking }) => {
+    onSubmit({
+      dismissed: true,
+      tracking,
+      marketing: tracking,
+      timestamp: Date.now(),
+    });
+
+    if (!tracking && defaultModel.tracking) {
```
Comment (suprMax) [inline packages/components/src/cookies_form/index.jsx:31]: perhaps we should move reload also to the outside handler?

File: packages/components/src/cookies_banner/index.jsx (hunk 1 above)
Comment (atomiomi) [reply packages/components/src/cookies_banner/index.jsx:20]: it's still needs to render <Script />

File: packages/components/src/cookies_banner/index.stories.jsx (hunk 2 above)
Comment (atomiomi) [reply packages/components/src/cookies_banner/index.stories.jsx:10]: @peterhass pls create a ticket in favro. Need to merge this

File: packages/components/src/cookies_form/index.jsx (hunk 3 above)
Comment (atomiomi) [reply packages/components/src/cookies_form/index.jsx:31]: will be duplicated in each fe repo

File: packages/components/src/cookies_banner/index.jsx (hunk 1 above)
Comment (suprMax) [reply packages/components/src/cookies_banner/index.jsx:20]: perhaps take `Scrip` out of here? would remove 4 props including `dismissed` and peer dependency

File: packages/components/src/cookies_banner/index.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+const CookiesBanner = ({
+  className,
+  children,
+  content,
+  ...rest
+}) => (
+  <article {...rest} className={clsx(className, styles.root)}>
+    <div className={styles.wrapper}>
+      <Markdown
```
Comment (suprMax) [inline packages/components/src/cookies_banner/index.jsx:16]: can be a 1 liner

**Review by suprMax — APPROVED** [review]

**Pull request merged**